website/docs/r/powerbidedicated_capacity.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "PowerBIDedicated"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_powerbidedicated_capacity"
+sidebar_current: "docs-azurerm-resource-powerbidedicated-capacity"
+description: |-
+  Manages a PowerBIDedicated Capacity.
+---
+
+# azurerm_powerbidedicated_capacity
+
+Manages a PowerBIDedicated Capacity.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_powerbidedicated_capacity" "example" {
+  name                = "example-powerbidedicatedcapacity"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  sku                 = "A1"
+  administrators      = ["azsdktest@microsoft.com"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the PowerBIDedicated Capacity. Changing this forces a new resource to be created.
```

Comment:
```suggestion
* `name` - (Required) The name which should be used for this PowerBI Dedicated instance. Changing this forces a new resource to be created.
```

Reply:
It seems that I cannot rename this description as what you suggested since this resource is to create capacity which is the feature of the power bi dedicated.
See more details from here:
Tutorial: https://docs.microsoft.com/en-us/azure/power-bi-embedded/
Rest api: https://docs.microsoft.com/en-us/rest/api/power-bi-embedded/capacities/create
azure-rest-api-spec: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/powerbidedicated/resource-manager/Microsoft.PowerBIdedicated/stable/2017-10-01/powerbidedicated.json